opts/opts.go

```diff
@@ -22,6 +22,7 @@ var (
 	// is not supplied. A better longer term solution would be to use a named
 	// pipe as the default on the Windows daemon.
 	DefaultHTTPPort = 2375 // Default HTTP Port
+	DefaultTLSHTTPPort = 2376 // Default TLS encrypted HTTP Port
```

Comment:
Should we add a mention here that these ports are registered, and a link to http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=docker